data.json

```diff
@@ -368,6 +368,15 @@
     "dateAdded": "2018-10-23T10:00:46.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
   },
+  {
+    "name": "Strawberry",
+    "difficulty": "easy",
+    "description": "Strawberry stickers for any accepted PR in October",
+    "reference": "https://twitter.com/patrick91/status/1178952690300272640",
+    "image": "https://pbs.twimg.com/media/EFx7NaiW4AA0RSD.jpg",
```

Comment:
Let's zoom on this image:

![2019-10-16-010805_364x501_scrot](https://user-images.githubusercontent.com/8191198/66876257-76a0a480-efb1-11e9-9151-c8dfc7a42ebf.png)

```suggestion
    "image": "https://i.imgur.com/QatMBLf.jpg",
```